app/controllers/staff/users_controller.rb

```diff
@@ -27,6 +27,8 @@ def create
     @user.active = params[:user][:active]
     authorize @user
     @organisations = policy_scope(Organisation)
+    @service_owner = service_owner
+    @delivery_partners = delivery_partners
```

Comment:
How does this work to fix the error (I can't tell just by looking 😄)? Would saying how in the commit message help future us understand why this made things work? Or might it make sense to cover the case with a spec?

Reply:
I've had a stab at this but I'm not happy with it. Would appreciate taking a look at it with someone.

Reply:
Fundamentally a `#create` in a controller won't ever have a template of its own; it's more usual to `redirect_to` a place where success is represented or `render` a different template for failure (as is the case here), so setting `@things = stuff` is odd in a `#create` (including that `@organisations = policy_scope(Organisation)` on line 29!)

Reply:
@dragon-dxw and I discovered that we need to leave these where they are for now as there are two code paths with a `render :new`. It will be possible to move them in `devise-authentication` where `users#create` is somewhat simplified.